Sources/KIF/Additions/UIView-KIFAdditions.m

```diff
@@ -630,10 +630,15 @@ - (void)dragFromPoint:(CGPoint)startPoint toPoint:(CGPoint)endPoint steps:(NSUIn
 }
 
 - (void)dragFromPoint:(CGPoint)startPoint displacement:(KIFDisplacement)displacement steps:(NSUInteger)stepCount;
+{
+    [self dragFromPoint:startPoint displacement:displacement steps:stepCount isFromEdge:NO];
```

Comment:
This is kind of a breaking change, in that it would require code changes in existing tests if they had working edge gestures. As you'd called out in the issue, this has been broken since iOS 11, so seems reasonable IMO.